*: change "synchronize" to "replicate" for migration related description (#1203)
pingcap/docs: dev/how-to/troubleshoot/data-migration.md

>
> Currently, TiDB is not compatible with all the DDL statements that MySQL supports. See [the DDL statements supported by TiDB](/sql/ddl.md).

## Reset the data replication task

You need to reset the entire data replication task in the following cases:

> - `RESET MASTER` is artificially executed in the upstream database, which causes an error in the relay log replication.

Reviewer: ```suggestion
- `RESET MASTER` is accidentally executed in the upstream database.
```
